frontend/app/components/folder-form.js

```diff
@@ -43,6 +46,14 @@ export default class FolderForm extends BaseFormComponent {
     });
   }
 
+  presetTeam() {
+    let selectedTeam = this.navService.selectedTeam;
+
+    if ((selectedTeam !== undefined) && this.isNewRecord){
+      this.selectedTeam = selectedTeam
+    }
```

Comment:
Same stuff here. Maybe you can even just put it inside the constructor directly since you'll just be doing
```
if (this.isNewRecord) this.selectedTeam = this.navService.selectedTeam
```